tests/cypress/e2e/CMakeLists.txt

```
@@ -69,3 +69,6 @@ set_tests_properties(cypress/e2e/tests PROPERTIES DEPENDS cypress/e2e/view-cover
 
 add_cypress_e2e_test(build-configure)
 set_tests_properties(cypress/e2e/build-configure PROPERTIES DEPENDS cypress/e2e/tests)
+
+add_cypress_e2e_test(all-projects)
+set_tests_properties(cypress/e2e/build-configure PROPERTIES DEPENDS cypress/e2e/build-configure)
```

I assume that's a copy-paste typo here
```suggestion
set_tests_properties(cypress/e2e/all-project PROPERTIES DEPENDS cypress/e2e/build-configure)
```